libfprint/drivers/elan.c

```diff
@@ -322,6 +322,11 @@ static void elan_capture_run_state(struct fpi_ssm *ssm)
     break;
 	case CAPTURE_SUBMIT_IMAGE:
         elan_submit_image(dev);
+        /* need to report finger off and process pending events here
+         * NOTE: timeval of 0.0 has no effect */
+        elan_set_finger_present(dev, FALSE);
+        struct timeval tv = { .tv_sec = 0, .tv_usec = 1 };
+        fp_handle_events_timeout(&tv);
```

Comment:
Why do you need it here? No other driver calls fp_handle_events_timeout()

Reply:
This is old code so fp_handle_events_timeout() is no longer used. But I remember that I couldn't make some mode (enroll, I think... not sure) work properly without it.